docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -0,0 +1,295 @@
+---
+id: dual-region-operational-procedure
+title: "Dual-Region Operational Procedure"
+sidebar_label: "Dual-Region Operational Procedure"
+description: "The operational procedure concerning dual-region setups to recover from a region loss."
+---
+
+import Swip from './swip.jsx';
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import SwipItem from './swipItem';
+
+<!-- Failover -->
+
+import Three from './img/3.svg';
+import Four from './img/4.svg';
+import Five from './img/5.svg';
+import Six from './img/6.svg';
+import Seven from './img/7.svg';
+
+<!-- Fallback -->
+
+import Nine from './img/9.svg';
+import Ten from './img/10.svg';
+import Eleven from './img/11.svg';
+import Twelve from './img/12.svg';
+import Thirteen from './img/13.svg';
+import Fourteen from './img/14.svg';
+import Fifteen from './img/15.svg';
+
+## Introduction
+
+The operational procedure is a step-by-step guide on how to proceed in the case of a total region failure. Allowing you to temporarily restore functionality and ultimately do a full recovery to restore the dual-region.
+
+## Disclaimer
+
+:::danger
+
+- Customers must develop and test operational procedures in non-production environments based on the framework steps outlined by Camunda.
+- Before advancing to production go-live, it is essential for customers to validate these procedures with Camunda.
+
+:::
+
+## Procedure
+
+We don't differ between primary and secondary regions as the procedure is the same for either loss. We will focus on losing the secondary region (passive) while still having the primary region (active).
+
+You'll have to take care of DNS considerations by rerouting traffic to the functioning region, which are disregarded in the following.
+
+After identifying or considering a region as lost, you should ensure that it doesn't reconnect, as this will hinder a successful recovery during failover and failback execution. <!-- Again, from what point on is a region loss fatal -->
+
+We will first look at the failover procedure, which is responsible for temporarily recovering the operations to unblock those. Zeebe cannot export and process any new data as long as it can't export those to ElasticSearch.
+
+Afterwards, the failback procedure is responsible for recovering a region.
+
+### Failover
+
+<Tabs>
+  <TabItem value="step1" label="Step 1" default>
```

Comment:
I kept it as steps for now but added titles for each step.
UI wise it looks bad to use Tabs with a lot of text.
We can iterate on this and see how we like it.

Since the docs are versioned I don't see an issue with future refactorings. We're probably keeping 8.5 as is and iterate further on next (8.6)